loky/backend/spawn.py

```diff
@@ -91,6 +91,7 @@ def get_preparation_data(name, init_main_module=True):
     # Figure out whether to initialise main in the subprocess as a module
     # or through direct execution (or to leave it alone entirely)
     if init_main_module:
+        d['init_main_module'] = True
```

Comment:
I am not opposed to this change but it seems completely unrelated to this PR.

Reply:
Also `d['init_main_module']` should be set to `False` in the `else:` branch of this condition and to add a test.

Reply:
I don't know why I did this changed here but it is useless.
The main module is initialized if one of the `init_main_from_name` and `init_main_from_path` is passed and this is already tested.
I will remove it, thanks for catching this.